provision-osx/tasks/configure_buildfarm_node.yml

```diff
@@ -0,0 +1,139 @@
+---
+
+-
+  debug:
+    msg:
+    - Configuring macOS nodes as part of the build farm will be skipped.
+    - credential_private_key_path, credential_public_key_path and credential_passphrase must be defined.
+    - Run this job again to configure the build farm nodes.
+  when:
+  - credential_private_key_path is not defined or credential_public_key_path is not defined or credential_passphrase is not defined
+
+-
+  block:
+  -
+    name: Get jenkins url
+    shell: "oc get route jenkins --namespace={{ project_name }} -o jsonpath='{.spec.host}'"
```

Comment:
I think `/vars/buildfarm.yml` (from root of the repo). Other roles have it specified as `jenkins` but then it gets overridden. I will add it into the defaults of this role though so it can run independent of this repo.